lib/playground.dart

```diff
@@ -931,10 +931,10 @@ class Playground implements GistContainer, GistController {
     _leftConsole.showOutput(message, error: error);
     _rightConsole.showOutput(message, error: error);
 
-    // If there's no tabs visible or the console is not being displayed,
-    // increment the counter
+    // If there are no tabs visible or the console is not being displayed,
```

Comment:
I applaud your attention to grammar and punctuation.

Reply:
Haha, when I read things out loud, I sometimes find little things like this.